modules/heat_conduction/test/tests/code_verification/cartesian_test_no2.i

```diff
@@ -0,0 +1,86 @@
+# Problem I.2
+#
+# The thermal conductivity of an infinite plate defined varies linearly
```

Comment:
An infinite plate with a thermal conductivity that varies linearly with temperature?